CHANGELOG.md

```diff
@@ -4,34 +4,8 @@
 
 ### Breaking Changes
 - Azure Communication Common Library
- - Communication identifier `MicrosoftTeamsUserIdentifier` property `identifier` renamed to `userId` since identifier was too generic.
- 
-## 1.0.0-beta.7 (2021-01-12)
```

Comment:
I think we should keep both changes here. 
You are missing the `1.0.0-beta.7 (2021-01-12)` change set

Reply:
thanks!! updated